### PR TITLE
DDT-784: GET transport document summaries

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java [hunk 1]
```diff
@@ -1,40 +1,182 @@
...
+                        return Mono.just(transportDocumentSummary);
+                      });
+            });
+  }
+
+  private Mono<Booking> getBooking(String carrierBookingReference, String shippingInstructionID) {
+    // Don't use ServiceClass - use Repository directly in order to throw internal error if
+    // BookingReference does not exist.
+    return bookingRepository
+        .findByCarrierBookingRequestReference(carrierBookingReference)
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java:169]: Why can't org.dcsa.core.events.edocumentation.service.ShipmentService#findByShippingInstructionID be used in addition with a switchIfEmpty()

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java [hunk 2]
```diff
@@ -1,40 +1,182 @@
...
+    }
+  }
+
+  @Transactional
+  @Override
+  public Mono<TransportDocumentTO> findById(String transportDocumentReference) {
+    return Mono.empty();
+  }
+
+  private Mono<Void> updateTransportDocumentWithCharges(
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java:104]: this method is not used

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java [hunk 3]
```diff
@@ -1,40 +1,182 @@
...
+      TransportDocumentTO transportDocumentTO, boolean isChargesDisplayed) {
+    return Mono.empty();
+  }
+
+  private Mono<Void> updateTransportDocumentWithTransportPlan(
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java:109]: this method is not used

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java [hunk 4]
```diff
@@ -1,40 +1,182 @@
...
+   * @param transportDocumentTO the TransportDocument to update
+   * @return Error if a discrepancy is found otherwise returns the Booking
+   */
+  private Mono<Booking> updateTransportDocumentWithBookingInfo(
+      String carrierBookingReference, TransportDocumentTO transportDocumentTO) {
+
+    return Mono.empty();
+  }
+
+  private <T> Mono<T> getBookingError(String fieldName, String fromValue, String toValue) {
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java:67]: this method does not seem to be used

**Review by pimg — CHANGES_REQUESTED** [review]

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java (hunk 1 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java]: method was unused, so I've removed it.

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java (hunk 3 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java]: removed

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java (hunk 2 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java]: removed

File: src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java (hunk 4 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java]: true, it's been removed

**Review by pimg — APPROVED** [review]

**Pull request merged**